### DISCUSSION TITLE
Using the Lighthouse Positioning System on Turtlebots and Zumos

### DISCUSSION THREAD

**Discussion by coopfpv** [bitcraze/discussions#867] [Q&A] [answered]
Hello,

I am hoping to gain some insight into whether the Lighthouse Positioning System must be used with a Crazyflie drone, or if the positioning decks are able to be mounted on robots such as the Turtlebot or the Zumo for pose tracking. I see on the site that the deck does firmware updates vie the Crazyflie, which made me wonder if it is possible to use this positioning system on vehicles other than the Crazyflie.

Thanks!

**Comment by krichardsson** [comment, marked as answer]
Hi, 
it is unfortunately not possible to (easily) use a Lighthouse deck on a different platform. Most of the positioning processing is done in the Crazyflie firmware and not on the deck itself. 

As all our code is opensource it is possible in theory to use it on another platform, but I would rate it as pretty complex.
